content/pt-br/docs/reference/glossary/toleration.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Tolerância
+id: toleration
+date: 2019-01-11
+full_link: /docs/concepts/scheduling-eviction/taint-and-toleration/
+short_description: >
+  Um objeto principal consistindo de três propriedades principais: chave, valor, e efeito. Tolerâncias habilitam o agendamento de pods em nós ou grupos de nós que tenham uma mancha coincidente. 
```

Comment:
It's better to use the original term for "taint", translating this here lost the meaning, what you think?

Reply:
Thanks, I updated this case as suggested, but kept the glossary references as `{{< glossary_tooltip text="manchas" term_id="taint" >}}`

Reply:
I agree with @edsoncelio that we should keep the word taint as is in this context. I suggest stylizing the word in italic so as to make it clear that this is a foreign word.

Reply:
@edsoncelio / @stormqueen1990

Should I replace {{< glossary_tooltip text="manchas" term_id="taint" >}} occurrences
by {{< glossary_tooltip text="taint" term_id="taint" >}} as well, or keep the translation in this case?

+1 to stylize the foreign words in italic! Thanks!

Reply:
Yes, please!

Reply:
@edsoncelio / @stormqueen1990 
Thanks, the suggestions were addressed!